paywall/src/data-iframe/cache/localStorage.js

```diff
@@ -43,6 +43,40 @@ function getContainer(window, key, type = false) {
   return container
 }
 
+/**
+ * Retrieve a cached value for a non-user-specific type on a specific network
+ *
+ * @param {object} window this is the global context, either global, window, or self
+ * @param {int} networkId the ethereum network id
+ * @param {string} type the type of account data to retrieve
+ */
+export async function getReadOnly(window, networkId, type) {
+  return get(
+    window,
+    networkId,
+    '0x0000000000000000000000000000000000000000',
```

Comment:
I'm going to add a comment in the code about why I chose address 0 (nobody can use this, so it's a guaranteed readonly value)